### PR TITLE
1.3 fixing

### PR DESCRIPTION
**Pull request by Arcadyi**

### CODE REVIEW

File: common/ai_strategies/03_political_strategies.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿ai_strategy_hardline_communist = {
+﻿1ai_strategy_hardline_communist = {
```
Comment (DZappala) [inline common/ai_strategies/03_political_strategies.txt:1]: Is this a typo?

File: common/buildings/01_industry.txt [hunk 2]
```diff
@@ -18,7 +18,16 @@ building_food_industry = {
...
 	}
 	required_construction = construction_cost_high
-
+	possible = {
+		error_check = {
+			severity = fail
+			this = {
+				owner = { 
+					NOT = { has_law = law_type:law_industry_banned } 
+				}
```
Comment (DZappala) [inline common/buildings/01_industry.txt:27]:
```suggestion
	possible = {
	    owner = {
           not = {
               has_law = law_type:law_industry_banned
           }
       }  
	}
```

We don't need to be error checking all of these, it's just a waste of processing power. Simply setting possible to weather or not they have the law is enough.

File: common/buildings/01_industry.txt
Comment (DZappala) [inline common/buildings/01_industry.txt:1]: See note, all should be changed.

File: common/buildings/06_urban_center.txt
Comment (DZappala) [inline common/buildings/06_urban_center.txt:1]: Same as note from industry.txt

File: common/character_traits/skill_traits.txt [hunk 3]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
-	
+
 	weight = {
 		value = 5
 		if = {
 			limit = {
 				commander_pm_usage = {
 					target = owner
-					production_method = pm_post_war_artillery
+					production_method = pm_mobile_artillery
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:245]: These should be our pms not the vanilla pms.

File: common/character_traits/skill_traits.txt [hunk 4]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
 			}
 			multiply = 5
 		}
 		else_if = {
-			limit = { 
+			limit = {
 				commander_pm_usage = {
 					target = owner
-					production_method = pm_self_propelled_artillery
+					production_method = pm_shrapnel_artillery
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:255]: same as prior

File: common/character_traits/skill_traits.txt [hunk 5]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
-			}
-			multiply = 15
-		}
-		else_if = {
-			limit = { 
+			limit = {
 				commander_pm_usage = {
 					target = owner
-					production_method = pm_guided_missile_artillery
+					production_method = pm_siege_artillery
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:265]: same as prior

File: common/character_traits/skill_traits.txt [hunk 6]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
+
 	weight = {
 		value = 5
 		if = {
-			limit = { 
+			limit = {
 				commander_pm_usage = {
 					target = owner
-					production_method = pm_mlrs_artillery
+					production_method = pm_shrapnel_artillery
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:301]: same as prior

File: common/character_traits/skill_traits.txt [hunk 7]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
 			}
 			multiply = 10
 		}
 		else_if = {
-			limit = { 
+			limit = {
 				commander_pm_usage = {
 					target = owner
-					production_method = pm_guided_missile_artillery
+					production_method = pm_siege_artillery
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:311]: same as prior

File: common/character_traits/skill_traits.txt [hunk 8]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
 		character_battle_condition_surprise_maneuver_mult = 0.5
 	}
-	
+
 	possible = {
 		has_role = general
-		owner = {
-			has_technology_researched = interwar_military_theory
+		owner ?= {
+			has_technology_researched = field_works
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:339]: this tech is vanilla

File: common/character_traits/skill_traits.txt [hunk 9]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
 		character_battle_condition_surprise_maneuver_mult = 1.5
 	}
-	
+
 	possible = {
 		has_trait = stalwart_defender
-		owner = {
-			has_technology_researched = modern_professional_army
+		owner ?= {
+			has_technology_researched = trench_works
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:370]: this tech is vanilla

File: common/character_traits/skill_traits.txt [hunk 10]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
 		character_battle_condition_surprise_maneuver_mult = 3
 	}
-	
+
 	possible = {
 		has_trait = trench_rat
-		owner = {
-			has_technology_researched = guerilla_warfare
+		owner ?= {
+			has_technology_researched = defense_in_depth
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:409]: this pm is vanilla

File: common/character_traits/skill_traits.txt [hunk 11]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
 					commander_pm_usage = {
 						target = owner
-						production_method = pm_7th_generation_infantry
+						production_method = pm_trench_infantry
 						value > 0.5
 					}
 					commander_pm_usage = {
 						target = owner
-						production_method = pm_8th_generation_infantry
+						production_method = pm_squad_infantry
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:426]: vanilla

File: common/character_traits/skill_traits.txt [hunk 12]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
 						value > 0.5
 					}
 					commander_pm_usage = {
 						target = owner
-						production_method = pm_9th_generation_infantry
+						production_method = pm_mechanized_infantry
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:431]: vanilla

File: common/character_traits/skill_traits.txt [hunk 13]
```diff
@@ -33,757 +34,645 @@ resupply_commander = {
...
 		character_battle_condition_surprise_maneuver_mult = 0.5
 	}
-	
+
 	possible = {
 		has_role = general
-		owner = {
-			has_technology_researched = combined_arms_warfare
+		owner ?= {
+			has_technology_researched = napoleonic_warfare
```
Comment (DZappala) [inline common/character_traits/skill_traits.txt:459]: vanilla

**Review by DZappala — CHANGES_REQUESTED** [review]

File: common/ai_strategies/03_political_strategies.txt (hunk 1 above)
Comment (Arcadyi) [reply common/ai_strategies/03_political_strategies.txt:1]: Err, that's not a vanilla one, probably a typo

**Pull request merged**